docs/release-notes/2.28.0.md

```diff
@@ -0,0 +1,108 @@
+# Percona Monitoring and Management 2.28.0
+
+| **Release date:** | May 12, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of the PMM instance. This ensures that you access the latest features of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+### Advisors
+
+- **Advisor checks enabled by default**
+
+  Starting with the previous release and continuing with this one, we have added significant improvements to the Advisors Checks functionality in performance, assessment coverage, and user experience.
```

Comment:
this looks weird on Render - it is not aligned with bullet list, so maybe we need to shift text to the right for all bullet list descriptions ?

Reply:
Sure @denisok . I will do a final check and take care of the alignment issues.